patches/minecraft/net/minecraft/entity/passive/HorseArmorType.java.patch

```diff
@@ -34,14 +34,18 @@
      public static HorseArmorType func_188580_a(ItemStack p_188580_0_)
      {
 -        return p_188580_0_.func_190926_b() ? NONE : func_188576_a(p_188580_0_.func_77973_b());
-+        return p_188580_0_.func_77973_b().getHorseArmorType(p_188580_0_);
++        if(p_188580_0_.func_77973_b() instanceof net.minecraftforge.common.IHorseArmor)
```

Comment:
Interface, makes absolutely no sense to be a cap and nothing will be gained from it being a cap.